db/migrate/20170313192244_create_new_db_schema_with_generic_page_version_change_annotation.rb

```diff
@@ -1,10 +1,11 @@
 class CreateNewDbSchemaWithGenericPageVersionChangeAnnotation < ActiveRecord::Migration[5.0]
   def change
-    # Postgres needs this for proper UUID support
-    enable_extension 'uuid-ossp'
+    # Since 9.4, PostgreSQL recommends using `pgcrypto`'s `gen_random_uuid`
+    # https://github.com/rails/rails/commit/b915b11cca558eb99b7c2621c4457491d4bdb43b
+    enable_extension 'pgcrypto'
```

Comment:
Is it safe to *remove* `uuid-ossp` here? (I actually don’t know.) I’m worried we’ll be missing something if we do. My `schema.rb` winds up having the `uuid-ossp` line removed, too, which also seems different from what you’re experiencing :\